widgets/build.gradle

```diff
@@ -35,6 +35,9 @@ dependencies {
 
     implementation kotlinn.stdlibjdk8
     implementation dagger.dagger
+    implementation androidx.appcompat
+    implementation androidx.constraintlayout
+    implementation multidex.multidex
```

Comment:
Let's set minSdkVersion = 21, in all modules. After that we don't need to add multidex implementation. Only multiDexEnabled true.
https://developer.android.com/studio/build/multidex#mdex-gradle

There is a variable minSdkVersion in project's build.gradle. Change it from 20 to 21. 
It is used in all build.gradle files of modules, but i forgot about it in widgets module. It's a mistake. Fix, please.

Reply:
Done!

Reply:
@MauzerTheCat 
What about implementation multidex.multidex ? We probably can delete it now.

Reply:
Yes, I already had... No idea, why it's shown here...